modules/grafana/manifests/dashboards/deployment_dashboard.pp

```diff
@@ -32,23 +32,39 @@
   $has_workers = false,
   $error_threshold = 20,
   $warning_threshold = 10,
+  $show_controller_errors = true,
+  $show_slow_requests = true
 ) {
   if $has_workers {
     $worker_row = [['worker_failures', 'worker_successes']]
   } else {
     $worker_row = []
   }
 
+  if $show_controller_errors and $show_slow_requests {
```

Comment:
Could we put these graphs into separate dashboard rows? It would make this logic simpler and I don't think it would change the appearance much.